benchmark/very_custom.yml

```diff
@@ -0,0 +1,23 @@
+# this is a heavily customized model used to benchmark routing with a custom weighting
+speed_factor:
+  road_class:
+    motorway: 0.85
+    primary: 0.9
+
+priority:
+  max_height: {"<3.8": 0}
+  max_width: {"<2.5": 0}
+  road_class:
+    motorway: 1
+    primary: 0.5
+    "*": 0.9
+  toll:
+    no: 1
+    "*": 0.5
+  hazmat:
+    no: 0
+
+max_speed_fallback: 110
+# todonow: ?
+# we have the other custom profile for speed comparison to "fastest" so no need to avoid distance influence here
+# distance_influence: 0
```

Comment:
I copy&pasted this from `Measurement.java`, is this still needed? I did not understand what the comment is supposed to say.

Reply:
It is not necessary anymore.